src/alertsmanagement/README.md

```diff
@@ -0,0 +1,46 @@
+# Azure CLI alertsmanagement Extension
+
+This extension can manage alert related resources. Currently, it supports
+action rule management.
+
+### How to use
+Install this extension using the below CLI command
+```
+az extension add --name alertsmanagement
```

Comment:
should this be released as monitor extension?

Reply:
There is not a central extension called monitor. Usually the name of the extension should be same with the command like `az extension add --name action-rule`. But it's indeed a general problem for all extensions. For those commands in extension, users don't know which extension package they should install.

Reply:
i see. Suggest to build a monitor extension maybe not now, I believe there will more resources added as monitor commands, from alerts management or Analytics etc. It will cause bad user experience.

Reply:
If using monitor as name, it will duplicate with monitor module. My suggestion is that move it into CLI core repo, abandon this extension.